src/content/blog/2024-06-20-simple-open-payments-guide.md

```diff
@@ -0,0 +1,90 @@
+
+This is how the Open Payments standard operates. Similar to an email address, if an account provider implements the Open Payments standard, they will provide you with a wallet address. This wallet address is human-readable (consisting of words instead of a long string of numbers) and publicly shareable (unlike your credit card number).
+
+When you want to make a payment online, you simply provide the application with your wallet address. This wallet address is also a URL, allowing the application to make queries directly to the wallet address. The application can then access information about where to contact your account provider. Using the Open Payments standard, the application can send a request directly to your account, asking for the specified amount to be transferred from your account to the recipient's account.
+
+As an open standard, Open Payments is free for anyone to use, ensuring seamless communication between applications and their user’s account providers, regardless of which financial institutions are involved. This direct communication means you're not incurring additional fees or delays.
+
+The Open Payments protocol also aims to make transactions more transparent. Before committing to a payment, you understand how much the recipient is receiving and how much this payment costs you in transaction fees.
+
+The Open Payments standard addresses key questions such as:
+
+- Where should the money be sent, exactly? It provides a standardized way to identify and locate user accounts across different financial institutions.
+- Ensuring all parties agree on the final amount after any fees are added to a transaction.
+- Determining the best underlying method to use for the transaction.
+
+With the Open Payments standard, applications do not need to be registered financial service providers (FSPs) to facilitate transactions. Instead of moving or holding money, applications send payment obligations. This means they are not transferring money directly but are sending instructions to your financial institution to transfer the funds on your behalf. This setup eliminates the need for the application to be a financial service provider itself, even though it has direct communication access to your account. This is beneficial for applications because they don’t need to contend with the legal and compliance hurdles required to be a financial services provider. But it’s also advantageous for you because you don't need to share sensitive information with online applications. Applications storing information like credit card numbers and CVV codes always pose a risk of data leaks or hacking, even with the best intentions.
```

Comment:
"But it’s also advantageous for you because you don't need to share sensitive information with online applications"

This is just a thought I had (especially knowing the banking context in Canada/North America), but we could also mention that without any API standards, some third-party apps like budget trackers rely on web-scraping to actually connect your bank accounts, meaning they end up knowing your banking login details too.